### DISCUSSION TITLE
Team Class

### DISCUSSION THREAD

**Discussion by Dumie101** [cwendt94/espn-api#163] [Q&A] [answered]
I created by League class. I am now wondering how to get the rosters of each team in my fantasy league. I am new to APIs and I don't understand how to create the team class that would allow me to get the roster.

**Comment by noah-kuo** [comment, marked as answer]
There is some example usage on the [Wiki](https://github.com/cwendt94/espn-api/wiki/Football-Intro) that I believe does what you're asking for. The league class should have a teams attribute that has all the team objects in a list.

**Reply by cwendt94** [reply]
Thanks @noah-kuo for answering! @Dumie101, the [Wiki](https://github.com/cwendt94/espn-api/wiki/Football-Intro) is a great start to understand all the features the package has to offer. If you have any other questions or can't find something on the Wiki please feel free to post!
